index.ios.js

```diff
@@ -1,3 +1,4 @@
 import App from 'components/App';
 
 const app = new App();
+console.disableYellowBox=true;
```

Comment:
remove this, even though you won't see any warnings on production, it's nice to have it by default to check errors/warnings.